package.json

```diff
@@ -0,0 +1,66 @@
+{
+	"name": "lundium",
+	"version": "0.1.0",
+	"private": true,
+	"description": "Collection of UI components used in Lundegaard projects",
+	"main": "index.js",
+	"repository": "https://MaceM8@bitbucket.org/MaceM8/lundium.git",
+	"contributors": [
+		"Martin Macháček <martin.machacek@lundegaard.eu>"
+	],
+	"license": "MIT",
+	"workspaces": [
+		"packages/*",
+		"apps/*"
+	],
+	"scripts": {
+		"build": "webpack --mode=production",
+		"dev": "webpack-dev-server",
```

Comment:
1. we use "start" for default dev task
2 why there is webpack-dev-server? I think the default task should look like:
```
"start": "yarn storybook start"
```